gui/wxpython/mapdisp/toolbars.py

```diff
@@ -241,16 +241,19 @@ def OnSelectTool(self, event):
         elif tool == self.toolId['3d'] and \
                 not (self.parent.MapWindow3D and self.parent.IsPaneShown('3d')):
             self.ExitToolbars()
+            self.parent.toolbars['map'].combo.SetValue(_("3D view"))
```

Comment:
Why not self.combo.SetValue?

Reply:
Fixed in the [7bd18fb](https://github.com/OSGeo/grass/pull/1198/commits/7bd18fbc94899b03d46f0db1dbcfd3c5b0e149da).